src/import/download_objectstore_files.py

```diff
@@ -75,9 +78,10 @@ def get_grondexploitatie_files():
 
             log.debug('AGE: %d %s', delta.days, expected_file)
 
-            if delta.days > 180:
-                log.error('DELEVERY IMPORTED FILES ARE TOO OLD!')
-                raise ValueError
+            if max_age > 0:
+                if delta.days > max_age:
+                    log.error('DELIVERY IMPORTED FILES ARE TOO OLD!')
+                    raise ValueError
```

Comment:
`raise ImportError` or something of the sort? ValueError seems out of place

Reply:
ImportError is voor het  de python import van modules die faalt. Die heeft een ander gedrag. Van de standaard excepties lijkt mij ValueError nog het meest toepasselijk.